sematic/cli/main.py

```diff
@@ -2,6 +2,11 @@
 from sematic.config.config import switch_env
 from sematic.db.migrate import migrate_up
 
+# Configure logging so that the migration container shows output
```

Comment:
We actually probably don't want the migration data showing when you don't explicitly run `sematic migrate`. The `migrate_up` gets executed on *every* CLI command, and seeing that every time would be distracting for people. Instead, let's:

- move the `migrate_up` call to the function `sematic/cli/cli:cli`. I believe putting it in there means it will get called before all sub-commands, but you can verify
- have `sematic/cli/cli:cli` take a `--verbose` flag which sets `logging.basicConfig(level=logging.INFO)`
- run the command in the K8s job as `sematic --verbose migrate`